community/event_management/doctype/event_details/event_details.json

```diff
@@ -0,0 +1,68 @@
+{
+ "actions": [],
+ "autoname": "field:event_name",
+ "creation": "2021-08-11 10:05:41.072432",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "event_name",
+  "start_date",
+  "end_date",
+  "event_details"
+ ],
+ "fields": [
+  {
+   "fieldname": "event_name",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Event Name",
+   "reqd": 1,
+   "unique": 1
+  },
+  {
+   "fieldname": "start_date",
+   "fieldtype": "Date",
+   "in_list_view": 1,
+   "label": "Start Date",
+   "reqd": 1
+  },
+  {
+   "fieldname": "end_date",
+   "fieldtype": "Date",
+   "in_list_view": 1,
+   "label": "End Date",
+   "reqd": 1
+  },
+  {
+   "fieldname": "event_details",
+   "fieldtype": "Markdown Editor",
+   "in_list_view": 1,
+   "label": "Event Details"
```

Comment:
Should we call this Event Description?

Reply:
Yes. Didn't pay attention. Will change it.